code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -11,58 +11,20 @@
 * Metal
 */
 GLOBAL_LIST_INIT(metal_recipes, list ( \
```

Comment:
Imo, make everything uselessl in a big recipe list (chairs, bed, and keep everything), semi usefull thing in another recipe list (wall grider, apc frame, metal rod, grenade casing etc.) in another recipe and the things you need in emergency diretly in the metal recipe (cades, wires)

Reply:
So don't remove any recipes?